diff --git a/CMakeLists.txt b/CMakeLists.txt
index a4e487b..a5b88fc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -97,12 +97,10 @@ endif()
 # And our helper modules
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 
-# We need boost
+# We need boost (system is header-only in Boost >=1.69, dropped here)
 find_package(Boost 1.41.0 REQUIRED COMPONENTS
   program_options
-  system
   thread
-  unit_test_framework
 )
 
 # Define where include files live
